spec/rubocop/ast/if_node_spec.rb

```diff
@@ -434,6 +434,13 @@
       it { expect(if_node.branches).to all(be_literal) }
     end
 
+    context 'with a ternary operator' do
+      let(:source) { 'foo? ? :foo : 42' }
+
+      it { expect(if_node.branches.size).to eq(2) }
+      it { expect(if_node.branches).to all(be_literal) }
```

Comment:
FYI, I think you can do both in a single expectaction `to match [be_literal, be_literal]`

Reply:
I just stick to the existing codestyle and replicated the nearest similar example 😄